src/FusionAuth/Provider.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace SocialiteProviders\FusionAuth;
+
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'FUSIONAUTH';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'openid',
+        'profile',
+        'email',
+    ];
+
+    protected $scopeSeparator = ' ';
+
+    protected function getFusionAuthUrl()
+    {
+        return $this->getConfig('base_url').'/oauth2';
```

Comment:
Potentially worth throwing an exception here if `base_url` is null? Otherwise will get a confusing string concat error.